src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -665,8 +666,8 @@ const WorkspaceDashboard = _.flow(
               Link,
               {
                 style: { marginLeft: '0.5rem' },
-                disabled: !!editWorkspaceError(workspace),
-                tooltip: editWorkspaceError(workspace) || 'Edit description',
+                disabled: !canEdit,
+                tooltip: editErrorMessage || 'Edit description',
```

Comment:
I think this would be slightly clearer as:

```suggestion
                tooltip: canEdit ? 'Edit description' : editErrorMessage,
```

Reply:
Agreed.